_data/projects/search/releases/6.2/6.2.0.CR1.yml

```diff
@@ -0,0 +1,10 @@
+date: 2023-06-26
+announcement_url: https://in.relation.to/2023/06/26/hibernate-search-6-2-0-CR1
+summary: >-
+  New `excludePaths` filter for `@IndexedEmbedded`,
```

Comment:
hmm, looks like it doesn't want to treat content from this yml as asciidoctor:
![image](https://github.com/hibernate/hibernate.org/assets/4004823/2f80d38f-99a5-4116-8e1f-58b7479e4d22)

Reply:
Yes, I'll need to work on that. It's not a new problem though :)

Reply:
I fixed that.